src/steps/findings.ts

```diff
@@ -26,7 +26,7 @@ export async function fetchFindings({
   await apiClient.iterateFindings(async (finding) => {
     const findingProps = finding.resource;
     let openBoolean: boolean;
-    findingProps.state === 'fixed'
+    findingProps.state === 'fixed' || findingProps.state === 'wont_fix'
       ? (openBoolean = false)
       : (openBoolean = true);
```

Comment:
It would be more idiomatic to go with:
```ts
const open = findingProps.state !== 'fixed' && findingProps.state !== 'wont_fix';
```
That said, you could even simplify with the following, to eliminate the need for comments later on:
```ts
createIntegrationEntity({
        entityData: {
          source: finding,
          assign: {
            ...
            open: findingProps.state !== 'fixed' && findingProps.state !== 'wont_fix',
            ...
          }
```

Reply:
Took the simplifcation.